zebrad/Cargo.toml

```
@@ -36,15 +36,15 @@ tracing-flame = "0.1.0"
 tracing-journald = "0.1.0"
 tracing-subscriber = { version = "0.2.25", features = ["tracing-log"] }
 tracing-error = "0.1.2"
-metrics = "0.13.0-alpha.8"
-metrics-exporter-prometheus = "0.1.0-alpha.7"
+metrics = "0.17.0"
+metrics-exporter-prometheus = "0.6.1"
 
 dirs = "4.0.0"
 inferno = { version = "0.10.7", default-features = false }
 atty = "0.2.14"
 
-sentry = { version = "0.21.0", default-features = false, features = ["backtrace", "contexts", "reqwest", "rustls"] }
-sentry-tracing = { git = "https://github.com/kellpossible/sentry-tracing.git", rev = "f1a4a4a16b5ff1022ae60be779eb3fb928ce9b0f" }
+sentry = { version = "0.23.0", default-features = false, features = ["backtrace", "contexts", "reqwest", "rustls"] }
+sentry-tracing = "0.23"
```

Let's use full versions, to avoid duplicate dependencies.
```suggestion
sentry-tracing = "0.23.0"
```

---

Done.